src/Model/Suppression/Bounce/Bounce.php

```diff
@@ -16,105 +16,44 @@
  */
 class Bounce
 {
-    /**
-     * @var string
-     */
     private $address;
-
-    /**
-     * @var string
-     */
     private $code;
-
-    /**
-     * @var string
-     */
     private $error;
-
-    /**
-     * @var \DateTime
-     */
     private $createdAt;
 
-    /**
-     * @param string $address
-     */
-    private function __construct($address)
+    private function __construct()
     {
-        $this->address = $address;
-        $this->createdAt = new \DateTime();
     }
 
-    /**
-     * @return Bounce
-     */
-    public static function create(array $data)
+    public static function create(array $data): self
     {
-        $bounce = new self($data['address']);
+        $model = new self();
 
-        if (isset($data['code'])) {
-            $bounce->setCode($data['code']);
-        }
-        if (isset($data['error'])) {
-            $bounce->setError($data['error']);
-        }
-        if (isset($data['created_at'])) {
-            $bounce->setCreatedAt(new \DateTime($data['created_at']));
-        }
+        $model->address = $data['address'];
```

Comment:
What if `address` is not set?

Reply:
thought it's ment to break, that address is mandatory. Will fix it

Reply:
I dont think we should assume anything that comes from the API